src/parser/semantic_check.c

```diff
@@ -8194,7 +8194,7 @@ pt_check_create_entity (PARSER_CONTEXT * parser, PT_NODE * node)
   DB_OBJECT *db_obj, *existing_entity;
   int found, partition_status = DB_NOT_PARTITIONED_CLASS;
   int collation_id, charset;
-  bool found_reuse_oid = false;
+  bool found_reuse_oid = false, reuse_oid = true;
```

Comment:
How about this?

8197 reuse_oid = true --> dont_reuse_oid_checked = false
...
8219 ~ 8237 lines --> like below
       case PT_TABLE_OPTION_REUSE_OID:
          {     
            if (found_reuse_oid || dont_reuse_oid_checked)
              {     
                PT_ERRORmf (parser, node, MSGCAT_SET_PARSER_SEMANTIC, MSGCAT_SEMANTIC_DUPLICATE_TABLE_OPTION,
                            parser_print_tree (parser, tbl_opt));
                return;
              }     
            else  
              {     
                found_reuse_oid = true; 
              }     
          }     
          break;

        case PT_TABLE_OPTION_DONT_REUSE_OID:
          {     
              if (found_reuse_oid)
              {     
                PT_ERRORmf (parser, node, MSGCAT_SET_PARSER_SEMANTIC, MSGCAT_SEMANTIC_DUPLICATE_TABLE_OPTION,
                            parser_print_tree (parser, tbl_opt));
                return;
              }     
              else
              {
                  found_reuse_oid = false;
                  dont_reuse_oid_checked = true;
              }
          }
...
8370 <-- not need to modify